src/e2e/resources/test.template.properties

```diff
@@ -70,7 +70,19 @@ test.timeout=15
 test.persistence.timeout=16
 
 ###############################################################################
+# The Google account of a user that has 'admin access' to the application.
+# This is the account used to login when testing against staging server.
+# It is also used as the main user whenever login is required for testing.
+
+test.admin.id=yourGoogleId
+
 # The email address below will receive some emails from the production server,
 # which will be programmatically checked as part of E2E tests.
 
 test.email=alice.tmms@example.tmt
```

Comment:
This should be changed to `@gmail.tmt`. Similarly for the `travis` and `travis-chrome` ones (although they won't affect anything).